### PR TITLE
feat: added default test cased from official Vue repo

### PR DESCRIPTION
**Pull request by juzhiyuan**
Add default test cases from official Vue repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,10 +55,12 @@
     "@vue/test-utils": "^1.0.0-beta.29",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.2",
+    "chromedriver": "78",
```
Comment (juzhiyuan) [inline package.json]: @moonming  We have some new dependencies for E2E test cases, so if I need to provide the License for them?

Comment (moonming) [reply package.json]: I think you can discuss license issues in mailing list.
 IMO, we only need add license for bundle codes in repo, not dependencies.

Comment (moonming) [reply package.json]: it's not need to add license if dependencies not in release packaege.

Comment (juzhiyuan) [reply package.json]: I see

**Pull request merged**